scripts/update_vllm_version.py

```diff
@@ -0,0 +1,46 @@
+# Standard
+import argparse
+import json
+import tomllib
+
+# Third Party
+import tomli_w
+
+
+def parse_args():
+
+    help_str = "Updates vllm version on pyproject.toml"
+    parser = argparse.ArgumentParser(description=help_str)
+    parser.add_argument(
+        "--vllm_version",
+        dest="vllm_version",
+        required=True,
+        type=str,
+        help="Version to overwrite pyproject.toml",
+    )
+    return parser.parse_args()
+
+
+def main():
+
+    args = parse_args()
+
+    print(f"Trying to update vllm package version to {args.vllm_version}:")
+
+    with open("pyproject.toml", "rb") as f:
+        data = tomllib.load(f)
+        data["project"]["optional-dependencies"]["vllm"][
```

Comment:
should we check and see if we are already at latest version here.. and the print `vllm already updated to latest version` and then exit without changing the file?